pkg/agent/nodeportlocal/k8s/npl_controller.go

```diff
@@ -137,10 +141,29 @@ func (c *NPLController) Run(stopCh <-chan struct{}) {
 	for i := 0; i < numWorkers; i++ {
 		go wait.Until(c.Worker, time.Second, stopCh)
 	}
-
+	c.SyncRules(stopCh)
 	<-stopCh
 }
 
+func NewTestingNPLController(portTable *portcache.PortTable) *NPLController {
+	// Only to be used in the Integration test and the Run method of NPLcontroller instance should not be called.
+	return &NPLController{
+		portTable: portTable,
+	}
+}
+
+func (c *NPLController) SyncRules(stopCh <-chan struct{}) {
+	f := func() {
+		if err := c.portTable.PodPortRules.SyncFixedRules(); err != nil {
+			return
+		}
+		klog.V(3).Infof("Periodic syncing of npl iptables")
+		_ = c.portTable.SyncRules()
```

Comment:
The blank identifier does not make sense here. If the return value of `SyncRules()` is always nil, you can remove the return value from the function. But I think it should be better to return the error at https://github.com/antrea-io/antrea/pull/3461/files#diff-b1c15dc1cb9b4dd9925e0f231ae82dedda9c89a25482da4be0a7ced5c5495ab4L437 and log it from the caller here.